activiti-api-impl/activiti-api-process-runtime-impl/src/main/java/org/activiti/runtime/api/impl/ProcessAdminRuntimeImpl.java

```diff
@@ -245,6 +261,9 @@ public void removeVariables(RemoveProcessVariablesPayload removeProcessVariables
     @Override
     @Transactional
     public void receive(ReceiveMessagePayload messagePayload) {
+        //@TODO: Check here, how to set processDefinitionKey
```

Comment:
you can probably remove the comment and open an issue

Reply:
Done